### ISSUE TITLE
Design Participant Query Component + Functionality

### ISSUE THREAD

**Issue by danoswaltCL** [CarnegieLearningWeb/UpGrade#1814]
Capturing what has been said in mtgs about a user feature-flag assignment "participant query" feature that Steve says he'd like to see used for feature-flags.

Potential Requirements:

- As an UI admin, I want a new root-level feature that will allow me to provide a `userId` and `context` and receive a list of all feature flags keys and experiment assignments that the user would be receiving in that environment, so that I can easily test to see if my feature-flags design setup is correct and I get expected keys from the browser.

- For example, on root page, I may see a nav item that will nav to a page with a form with a "userId" text input and a app-context dropdown. When I submit these two pieces of information, I will see some kind of display of the flag keys and assignments.

- Essentially this would be the exact same kind of behavior as the client call to `/featureflags` or `/assign by `userId` and `context`, but would want it to be a new UI admin endpoint that does not produce any side-effects like marking an exposure.

- Q: It would require that a flag / experiment be set to enrolling in order to simulate the "real-time" assignment... but I think the user will often want to use this feature to actually test what the user WOULD get if the flag was enabled, or the experiment was enrolling, so they can be more confident _before_ clicking that ON switch. That may be a slightly different requirement if so.

- If this is not implemented at root level, it could be a common component that could be reusable in both the feature-flags and experiments feature.

**Comment by SritterCL** [comment]
Yeah - I'm thinking this might also be useful for customer support (and, more generally, bug tracking). The "enrolling" issue is interesting, but for version 1, I'd just treat this as a feature that works when things are "live" (so it really does duplicate what "assign" would do).
